test/unit/model/model-test.js

```diff
@@ -35,30 +35,57 @@ describe('Model', function() {
     });
 
     describe('#_normalizeConfig', () => {
-        it('should leave the floating block untouched when disabled is not true', () => {
-            let startingCfg = {
-                floating: { disabled: false }
-            };
-            model._normalizeConfig(startingCfg);
-            expect(startingCfg.floating).to.deep.eq({ disabled: false });
+        describe('floating block', () => {
```

Comment:
only change to the floating block tests was scoping them to their own describe block